akka-docs/src/main/paradox/scala/guide/quickstart.md

```diff
@@ -1,136 +1,26 @@
 # Quickstart
```

Comment:
I agree with the point of having a landing page on the website that would be tailored for the first contact. 

But I am not sure whether have that *and* a similar page in the docs is needed. We should have only one place which we should recommend as a first contact experience.